pages/common/figlet.md

```diff
@@ -9,12 +10,8 @@
 
 - Use a custom font file:
```

Comment:
```suggestion
- Use a custom font file (full path  and extention can be omitted if the font file is in the default font directory):
```

Reply:
@marchersimon I added a new example for this.